prow/cmd/line/main.go

```diff
@@ -509,7 +509,7 @@ The magic incantation to run this job again is ` + "`%s`" + `. Please help us cu
 %s
 </details>
 `
-	body := fmt.Sprintf(bodyFormat, c.Presubmit.Context, url, c.PullSHA, c.PRNumber, c.Presubmit.RerunCommand, c.RepoOwner, c.RepoName, plugins.AboutThisBot)
+	body := fmt.Sprintf(bodyFormat, c.Presubmit.Context, url, c.PullSHA, c.PRNumber, c.Author, c.Presubmit.RerunCommand, c.RepoOwner, c.RepoName, plugins.AboutThisBot)
```

Comment:
this is starting to get complex enough that it might be worth using text/template?